WordPress/Classes/ViewRelated/Reader/ReaderTagStreamHeader.xib

```diff
@@ -1,110 +1,66 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The follow button IBAction is showing up as not connected properly – might just need disconnecting and reconnecting.